classes/import-export.php

```diff
@@ -514,6 +515,24 @@ public function import_csv() {
 		}
 
 		/**
+		 * Gets import path.
+		 *
+		 * @param $file
+		 *
+		 * @return string
+		 */
+		function get_import_path( $file ) {
+			$upload_dir = wp_upload_dir();
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Method name "SM_Import_Export::get_import_path" is not in camel caps format
* Visibility must be declared on method "get_import_path"
* Expected 0 spaces between opening bracket and argument "$file"; 1 found
* Expected 0 spaces between argument "$file" and closing bracket; 1 found
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->